### ISSUE TITLE
Should this start back up?

### ISSUE THREAD

**Issue by kingcodefish** [IndieCollab/Shadows-Of-Seven#3]
I know I slowed you guys down with the unfinished physics engine, but should this start progress again? I would be eager to, but I need Indie's permission first. I know he's a bit frustrated with the community ATM, trying to find ideas, but I was just trying to see if our work should see the light of day?

**Comment by IndieKA** [comment]
yeah sure, in fact, go ahead and release the game yourself when you are done.

**Comment by kingcodefish** [comment]
What, really? Why is that, I would like to work on it with at least you.

**Comment by IndieKA** [comment]
Sure, whatever. I'm disappointed with ka right now so I'll do the work on a new account. MathIM, or not?

**Comment by kingcodefish** [comment]
Sure, link please.

**Comment by IndieKA** [comment]
http://mathim.com/rektka99
